frontend/src/components/QuickFilters/FilterRenderers/Slider/Slider.tsx

```diff
@@ -0,0 +1,14 @@
+import './Slider.styles.scss';
+
+import { IQuickFiltersConfig } from 'components/QuickFilters/QuickFilters';
+
+interface ISliderProps {
+	filter: IQuickFiltersConfig;
+}
+
+// not needed for now build when required
+export default function Slider(props: ISliderProps): JSX.Element {
+	const { filter } = props;
+	console.log(filter);
```

Comment:
Can you remove this console statement? 
Rest LGTM.

Reply:
this is no where used , will start giving eslint errors for unused filters. will remove when we consume this.